sale_order_volume/__openerp__.py

```diff
@@ -19,7 +19,7 @@
 ##############################################################################
 {
     "name": "Sale Order Volume",
-    "version": "1.0",
+    "version": "1.1",
```

Comment:
we went for OCA versioning convention here is an example : https://github.com/OCA/crm/blob/9.0/crm_action/__openerp__.py#L8

Reply:
at line 27 could you replace in the author key Robin Keuken by Coop IT Easy SCRLfs as now the contributor appears in the header.

Reply:
indeed, we agreed on

```
# Copyright 2019 Coop IT Easy SCRLfs
# 	Robin Keunen <robin@coopiteasy.be>
# 	Rémy Taymans <remy@coopiteasy.be>
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).

{
    ...
    "author": "Coop IT Easy SCRLfs",
    ...
}
```

Reply:
Changes made !